aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move

```diff
@@ -1328,6 +1334,8 @@ module supra_framework::pbo_delegation_pool {
                 amount_unlocked: amount,
             },
         );
+        let (active_stake, _, pending_active, _)= stake::get_stake(pool_address);
+        assert!( active_stake + pending_active == pool_u64::total_coins(&pool.active_shares), error::invalid_state(ACTIVE_COIN_VALUE_NOT_SAME_STAKE_DELEGATION_POOL));
```

Comment:
@sjoshisupra looks this invariant currently works. I am not sure this invariant is correct or not.